### PR TITLE
close #373, needs manual intervention

### LINKED ISSUE #373: UPDATE Topics-Items
As a reporter (Björn)
I want a Topic List (in German!) as follows
in order to fit the editorial planing.

*TOPICS*
Breaking News
Hofleben
Abkalbung
Gesundheit
Backup-Kuh
Technologie
Kalb
Milchmenge
Milchqualität
Fütterung
Kuhalltag
Sozialverhalten
Brunst_und_Besamung
Politik_Ethik
Wirtschaft (Milchpreis)
Kurioses

### PR DESCRIPTION
**Pull request by roschaefer**
close #373

### CODE REVIEW

File: lib/tasks/add_german_topics.rake [hunk 1]
```diff
@@ -0,0 +1,43 @@
+namespace :sensorstory do
+
+  desc 'Add or rename a list of german topics'
+  task add_german_topics: :environment do
+    Topic.transaction do
+      not_matched = [
```
Comment (roschaefer) [inline lib/tasks/add_german_topics.rake]: @drjakob what happens with these topics?

Comment (drjakob) [inline lib/tasks/add_german_topics.rake]: 3 things: 1. Structure content (eg for search), 2. Control Chatbot, 3. Define Icons for Timestamps

Comment (roschaefer) [inline lib/tasks/add_german_topics.rake]: no, I mean, there are three topics that are not in the list of #373

Comment (roschaefer) [inline lib/tasks/add_german_topics.rake]:
```
    "movement",
    "temperature",
    "noise"
```

Comment (drjakob) [inline lib/tasks/add_german_topics.rake]: Just delete them from text components.

Comment (roschaefer) [inline lib/tasks/add_german_topics.rake]: Ok, `movement`, `temperature` and `noise` had 2, 3 and 3 text components.

File: db/schema.rb [hunk 2]
```diff
@@ -139,7 +139,6 @@
     t.float    "max_value"
     t.float    "min_value"
     t.datetime "calibrated_at"
-    t.boolean  "smaxtec_sensor"
```
Comment (tillprochaska) [inline db/schema.rb]: Is the removal of the `smaxtec_sensor` property intended? Not sure how this is related to issue #373…

Comment (drjakob) [inline db/schema.rb]: As far as I know rhe property is given when an animal id is present

Comment (roschaefer) [inline db/schema.rb]: @tillprochaska that's a good find @matthib the `schema.rb` was generated. So I believe this is a leftover of the schema.rb that was merge into master? Do we need to care about `smaxtec_sensor`?

**Pull request merged**